lib/engine/game/g_18_carolinas/step/track.rb

```diff
@@ -62,7 +62,7 @@ def revenue_fail
           end
 
           def process_run_routes(action)
-            hexes = action.routes[0].connection_hexes.flat_map.uniq { |h| @game.hex_by_id(h) }
+            hexes = action.routes[0].connection_hexes.flatten.uniq.map { |h| @game.hex_by_id(h) }.uniq
```

Comment:
why do you have two uniqs

Reply:
Uh, to make sure it's extra unique? I'll fix.